backend/utils/request_validation.py

```
@@ -0,0 +1,192 @@
+
+                # Add validated data to request context
+                request.validated_data = validated_data
+
+                return f(*args, **kwargs)
+
+            except ValidationError as e:
+                return (
+                    jsonify({"error": "Validation failed", "details": e.messages}),
+                    400,
+                )
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)